client/.eslintrc

```diff
@@ -18,6 +18,12 @@
    "react/react-in-jsx-scope": "off",
    "react/prop-types": "off",
    "react-hooks/rules-of-hooks": "error",
-   "react-hooks/exhaustive-deps": "error"
+   "react-hooks/exhaustive-deps": "error",
+   "prettier/prettier": [
+    "error",
+    {
+      "endOfLine": "auto"
+    }
+  ]
```

Comment:
I actually noticed this array not being indented properly which makes me think your prettier isn't automatically linting your files - may want to double check it's working. On any JS files, on save it should automatically lint your file to fix any spacing issues (that's just one thing it will lint)